spec/features/dams_unit_spec.rb

```diff
@@ -92,5 +97,11 @@
     
     # does not find this collection because it belongs to other unit
     expect(page).to_not have_selector('a', :text => 'Test Collection')
-  end  
+  end
+  
+  scenario 'rdcpt collection search' do
+    visit dams_unit_collections_path('rdcpt')
```

Comment:
@mcritchlow Using more general contextual search display text like "Search and browse contextually within a unit" sounds good to me. Are you suggesting using this contextual search display text just for the test unit here, or it should be displayed on the search box for any units?